openslides_backend/action/actions/user/send_invitation_email.py

```diff
@@ -148,31 +148,27 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             ],
         )
         if not (to_email := user.get("email")):
-            result["message"] = f"User/{user_id} has no email-address."
+            result["message"] = f"'{user['username']}' has no email address."
             return instance
         if not self.check_email(to_email):
-            result[
-                "message"
-            ] = f"The email-address {to_email} of User/{user_id} is not valid."
+            result["message"] = f"'{user['username']}' has no valid email address."
             return instance
         result["recipient"] = to_email
 
         if meeting_id and meeting_id not in user["meeting_ids"]:
             result[
                 "message"
-            ] = f"User/{user_id} does not belong to meeting/{meeting_id}"
+            ] = f"'{user['username']}' does not belong to meeting/{meeting_id}"
             return instance
 
         mail_data = self.get_data_from_meeting_or_organization(meeting_id)
         from_email: Union[str, Address]
         if users_email_sender := mail_data.get("users_email_sender", "").strip():
             blacklist = ("[", "]", "\\")
             if any(x in users_email_sender for x in blacklist):
-                result["message"] = (
-                    f'Invalid characters in the sender name configuration of meeting_id "{meeting_id}". Not allowed chars: "'
-                    + '", "'.join(blacklist)
-                    + '"'
-                )
+                result[
```

Comment:
Why you want to change this message? IMO the existing one is better, because more expressive

Reply:
ok, I'll change it back to integrate the meeting name and the forbidden characters. I had the client in mind which has trouble translating strings with variables in it, but with https://github.com/OpenSlides/openslides-backend/pull/1502 that should no longer be a problem